build.rs

```diff
@@ -49,3 +52,23 @@ fn expand_serde(out_dir: &Path) {
         serde_codegen::expand(&out_dir.join("tmp.rs"), &dst).unwrap();
     }).unwrap().join().unwrap();
 }
+
+// TODO: how to only do this for tests? I tried to do:
+//
+// > #[cfg(all(test, feature = "demangle_cpp"))]
+//
+// but it did not work...
+#[cfg(feature = "demangle_cpp")]
```

Comment:
This is unfortunate. Perhaps you know how we can only link the test C++ code when building for tests?